### PR TITLE
Update/afternoon presets

### PR DESCRIPTION
**Pull request by troychaplin**
**Description**

Fixing some of the items noted in #538

### CODE REVIEW

**Comment by karmatosed** [comment]
@troychaplin it looks like a few things would be helpful here:

1. Can you please provide all the fixes if possible for this in the PR so we can get them all in? Unless I am mistaken some are missing.
2. Can you show a screenshot of the changes please?

**Comment by troychaplin** [comment]
> @troychaplin it looks like a few things would be helpful here:
> 
>     1. Can you please provide all the fixes if possible for this in the PR so we can get them all in? Unless I am mistaken some are missing.
> 
>     2. Can you show a screenshot of the changes please?

More than happy to try, I made some comments on the [issue here](https://github.com/WordPress/twentytwentyfive/issues/538), would be happy to cycle back and try and finish them off with a bit of clarity on my notes. Might not get a chance to look until tomorrow, holiday weekend in Canada, away most of the day.

**Comment by troychaplin** [comment]
@karmatosed here are some before and after images of each change I've made.

--
Change font preset XXL: min 2rem, max 2.6rem & Change font preset XL: min 1.4rem, 1.8rem

font-preset-xxl-xl

--
Text: set to font-weight: 300, line height to 1,5

text-weight-height

--
Headings: set to font-weight: 400, letterSpacing: -0,1px , line-height: 1.2

heading-weight-space-height

--
Site Title: set to Large font size

site-title-large

--
Titles (Post Titles): any specific settings should be removed, they should inherit the headings settings

post-titles-remove-settings

**Comment by troychaplin** [comment]
@karmatosed In the json file for afternoon I changed the styles for `post-author-name` but now I wonder if `post-author` should also be changed? (maybe even `post-author-biography`?)

--
Another thing I noticed and want to make sure it's of no concern. When I remove the `post-author-name` in the afternoon json file it displays larger than I would expect in the style book, like the following:

removed-from-json

--
If I change it to small instead of removing it, the size displays correctly in the style book like this:

set-as-small

--
Regardless of the size display in the book, the front end is correct regardless of which change is made, I test both and they both look like the following:

frontend-output

**Comment by troychaplin** [comment]
@karmatosed I went back to the list of heading sizes and I think it clicked. I tested that each heading was set to the size in the original list by temporary changing the `fontSizes` under `settings` / `typography` and testing in the editor and style book.

In the process I added settings for h5 and h6 as per the original notes in issue #538 

The original list of requested changes are:

```
H1: XXL font size
H2: XL font size
H3: L font size
H4: M font size
H5: M font size
H6: S font size + (remove uppercase, letter spacing, remove specific font weight, it should be 400 same as others)
```

Adding styles for h5 and h6 looks like the following:

heading-sizes

**Comment by carolinan** [comment]
Correct, if the sizes match those that are in theme.json, they do not need to be added in these files, they will be inherited from theme.json.

**Comment by beafialho** [comment]
Thank you @troychaplin!

All is looking as expected to me.

> In the json file for afternoon I changed the styles for post-author-name but now I wonder if post-author should also be changed?

That would be ok, I think. I don't think that block is used in the templates, but if it has specific settings it could potentially behave awkwardly if users add it to templates so maybe that's a wise call.

**Comment by troychaplin** [comment]
> Thank you @troychaplin!
> 
> All is looking as expected to me.
> 
> > In the json file for afternoon I changed the styles for post-author-name but now I wonder if post-author should also be changed?
> 
> That would be ok, I think. I don't think that block is used in the templates, but if it has specific settings it could potentially behave awkwardly if users add it to templates so maybe that's a wise call.

Ok, will make this last change shortly!

**Comment by troychaplin** [comment]
@karmatosed @beafialho @carolinan I have removed `post-author`. I also took a look at `post-author-biography` and also remove it and now all the `post-*` referenced in the style book are consistent

File: styles/04-afternoon.json [hunk 1]
```diff
@@ -97,9 +97,9 @@
 	"styles": {
 		"typography": {
 			"fontFamily": "var:preset|font-family|ysabeau-office",
-			"fontWeight": "400",
+			"fontWeight": "300",
```
Comment (juanfra) [inline styles/04-afternoon.json]: This value is already coming from `theme.json`

```suggestion
```

File: styles/04-afternoon.json [hunk 2]
```diff
@@ -132,27 +132,19 @@
...
-			"core/list": {
+			"core/heading": {
 				"typography": {
-					"lineHeight": "1.3"
-				}
-			},
-			"core/loginout": {
-				"typography": {
-					"fontSize": "var:preset|font-size|medium"
+					"fontWeight": "400",
```
Comment (juanfra) [inline styles/04-afternoon.json]: This value is already coming from `theme.json`

```suggestion
```

File: styles/04-afternoon.json [hunk 3]
```diff
@@ -132,27 +132,19 @@
...
+					"letterSpacing": "-0.1px",
```
Comment (juanfra) [inline styles/04-afternoon.json]: This value is already coming from `theme.json`
```suggestion
```

**Review by juanfra — COMMENTED** [review]
Thank you very much @troychaplin - Looking good to me.

I left a few adjustments I'm going to commit now and then we're good to merge.

**Review by juanfra — APPROVED** [review]

**Pull request merged**